### PR TITLE
Add `CODE_OF_CONDUCT.md`

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This repository is a TC39 project and subscribes to its [code of conduct](https://tc39.es/code-of-conduct/). It is available at [https://tc39.es/code-of-conduct/](https://tc39.es/code-of-conduct/).
```
Comment (ctcpip) [inline CODE_OF_CONDUCT.md]:
```suggestion
This repository is a TC39 project and subscribes to its [Code of Conduct](https://tc39.es/code-of-conduct/). It is available at [https://tc39.es/code-of-conduct/](https://tc39.es/code-of-conduct/).
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+To ask a question or report an issue, please email [tc39-conduct-reports@googlegroups.com](mailto:tc39-conduct-reports@googlegroups.com).
```
Comment (ctcpip) [inline CODE_OF_CONDUCT.md:3]: not sure if we should provide this call to action here.  preferable for them to see the reporting guidelines on the CoC page.  thoughts @ljharb ?

**Review by ctcpip — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (ctcpip) [reply CODE_OF_CONDUCT.md:3]: update:  I see that's what's in `CODE_OF_CONDUCT.md` in one or more repos, so I guess consistency is good?

**Comment by ctcpip** [comment]
I'm seeing now this is copy/pasta from another repo, so this is fine.  We should update and normalize them as a separate activity.

**Review by ctcpip — APPROVED** [review]

**Review by ljharb — APPROVED** [review]
LGTM with the above comments

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (ljharb) [reply CODE_OF_CONDUCT.md:3]: I'm fine with it either way, personally

Comment (smorimoto) [reply CODE_OF_CONDUCT.md:3]: Ideally, I think, create a repository for the coc, and all repositories should have a minimal pointer to that repository (or more accurately, TC39 coc?).

Comment (ljharb) [reply CODE_OF_CONDUCT.md:3]: We have that repo already, and that's what this PR does, right?

**Pull request merged**